src/components/tests/content-tab-snapshots.test.tsx

```diff
@@ -474,4 +474,41 @@ describe( 'AddDemoSiteWithProgress', () => {
 		} );
 		expect( offlineTooltip ).toBeVisible();
 	} );
+
+	test( 'confirms that Clear expired site button is present on expired snapshot and calls removeSnapshot when clicked', async () => {
```

Comment:
I wonder if we could also cover in this test case that the demo site is not shown after removing the snapshot. WDYT?

Reply:
Makes sense! Let's add it

Reply:
I made some updates for it here: https://github.com/Automattic/studio/pull/218/commits/678d9068aa86f39151e9b891981b2d634648e77b